db/migrate/20191015145846_add_status_to_carriers.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToCarriers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :carriers, :status, :string, default: 'Disabled'
```

Comment:
@abronkema what would be the default for the carriers status? What does `disabled` mean in this context? That it's not enabled for selling?

Reply:
It would also be good for this column to be not null: https://stackoverflow.com/a/20255645/1279707